server.js

```diff
@@ -7395,6 +7395,39 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// PHP version from Packagist
+camp.route(/^\/php\/v\/packagist?\/([^/]+\/[^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var userRepo = match[1];  // eg, espadrine/sc
+  var version = match[2];
+  var format = match[3];
+  var options = {
+    method: 'GET',
+    uri: 'https://packagist.org/p/' + userRepo + '.json',
+  };
+  var badgeData = getBadgeData('PHP', data);
+  request(options, function(err, res, buffer) {
+    if (err != null) {
+      log.error('Packagist error: ' + err.stack);
+      if (res) {
+        log.error('' + res);
+      }
+      badgeData.text[1] = 'invalid';
+      sendBadge(format, badgeData);
+      return;
+    }
+
+    try {
+      var data = JSON.parse(buffer);
+      badgeData.text[1] = data.packages[userRepo][version].require.php;
+      badgeData.colorB = '#8892BF';
```

Comment:
Could you use `colorscheme = 'blue'` instead? That's what most of our version badges use right now.

Reply:
You didn't say anything against in #819.

This color is the color of the PHP brand.

<details>
<summary>screenshot</summary>

![image](https://user-images.githubusercontent.com/1954436/32593023-5273bb7a-c537-11e7-8052-45510a54297c.png)

</details>


If it's critical, i can use blue.

Reply:
Yes please. Providing consistent badges is one of the goals of the project.

Reply:
fixed